### PR TITLE
Update border-radius

### PR DESCRIPTION
**Pull request by viggo**
Update border-radius

Old:
border-radius: 2px

New:
border-radius: 2px;
border-radius-8: 8px;
border-radius-16: 16px;
border-radius-24: 24px;
border-radius-32: 32px;

Please double check that tw-tokens.json has the right format.

### CODE REVIEW

**Comment by juliuxu** [comment]
@viggo This change is for both the Posten and Bring brand?

**Comment by juliuxu** [comment]
As the code stands now both posten and bring will go from 2px to 8px by default for all radiuses. I thought this was just for the "new" PostenBring brand??

**Comment by viggo** [comment]
> As the code stands now both posten and bring will go from 2px to 8px by default for all radiuses. I thought this was just for the "new" PostenBring brand??

Yes, this is done purposefully:
https://www.figma.com/design/64ieYInJRps5diLI1qgJJk/Hedwig-Design-System?node-id=16258-6187&t=iEBJVXSk5QDaSb4x-1

File: apps/examples/app/examples/patterns/tailwind/demo.tsx
Comment (juliuxu) [inline apps/examples/app/examples/patterns/tailwind/demo.tsx:1]: Works with tailwind
image

**Review by juliuxu — APPROVED** [review]

**Pull request merged**